src/coreComponents/common/format/table/TableFormatter.hpp

```diff
@@ -138,7 +137,7 @@ class TableTextFormatter : public TableFormatter
   /**
    * @return A TableLayout converted into a formatted representation.
```

Comment:
Try to **explain** more than just describing the code:
```suggestion
   * @return the table with no data, so only the top part is returned. 
```
(something like that, suggestion may not be perfect)

Reply:
Please correct your correction.

Reply:
Please fix "The TableTextFormatter receives hasn't receive any data" sentence